cmd/open_other.go

```diff
@@ -7,24 +7,43 @@ import (
 	"io"
 	"os"
 
+	"github.com/google/go-tpm-tools/client"
 	"github.com/google/go-tpm/legacy/tpm2"
 )
 
 var tpmPath string
 
+// tpmWrapper wrap tpm io.ReadWriteCloser and EventLogGetter interfaces together.
+type tpmWrapper struct {
+	io.ReadWriteCloser
+	client.EventLogGetter
+}
+
+// EventLog allows caller to call the EventLogGetter function
+// of the wrapped TPM.
+func (et tpmWrapper) EventLog() ([]byte, error) {
+	if ExternalTPM != nil {
+		return client.GetEventLog(et.ReadWriteCloser)
+	}
```

Comment:
extTPMWrapper should be used when ExternalTPM is set. This should be used otherwise (see `openTpm`). We should not check ExternalTPM here.